xExtension-YouTube/configure.phtml

```diff
@@ -1,26 +1,29 @@
+<?php $this->loadConfigValues(); ?>
 
 <form action="<?php echo _url('extension', 'configure', 'e', urlencode($this->getName())); ?>" method="post">
 	<input type="hidden" name="_csrf" value="<?php echo FreshRSS_Auth::csrfToken(); ?>" />
 	<div class="form-group">
 
 		<label class="group-name" for="yt_height"><?php echo _t('ext.yt_videos.height'); ?></label>
 		<div class="group-controls">
-			<input type="number" id="yt_height" name="yt_height" value="<?php echo FreshRSS_Context::$user_conf->yt_player_height; ?>" min="50" data-leave-validation="1">
+			<input type="number" id="yt_height" name="yt_height" value="<?php echo $this->getHeight(); ?>" min="50" data-leave-validation="1">
 		</div>
 
 		<label class="group-name" for="yt_width"><?php echo _t('ext.yt_videos.width'); ?></label>
 		<div class="group-controls">
-			<input type="number" id="yt_width" name="yt_width" value="<?php echo FreshRSS_Context::$user_conf->yt_player_width; ?>" min="100" data-leave-validation="1">
+			<input type="number" id="yt_width" name="yt_width" value="<?php echo $this->getWidth(); ?>" min="100" data-leave-validation="1">
 		</div>
-
-		<label class="group-name" for="yt_show_content"><?php echo _t('ext.yt_videos.show_content'); ?></label>
+		
+		<label class="group-name" for="yt_content"><?php echo _t('ext.yt_videos.content.title'); ?></label>
 		<div class="group-controls">
-			<input type="checkbox" id="yt_show_content" name="yt_show_content" value="1" <?php if (FreshRSS_Context::$user_conf->yt_show_content != '') { echo 'checked'; }; ?>>
+			<input type="radio" id="yt_content_none" name="yt_content_choice" value="none" <?php if ($this->getContentHandling() === 'none') { echo 'checked'; }; ?>> <?php echo _t('ext.yt_videos.content.none'); ?><br />
+			<input type="radio" id="yt_content_append" name="yt_content_choice" value="append" <?php if ($this->getContentHandling() === 'append') { echo 'checked'; }; ?>> <?php echo _t('ext.yt_videos.content.append'); ?><br />
+			<input type="radio" id="yt_content_format" name="yt_content_choice" value="format" <?php if ($this->getContentHandling() === 'format') { echo 'checked'; }; ?>> <?php echo _t('ext.yt_videos.content.format'); ?><br />
```

Comment:
This part can be improved by placing the input and label inside a `<label>` tag so all the lines become clickable. Note I'm not sure it's good practice to place input in a label, but it's how it's done for checkboxes and radios elsewhere in FreshRSS code so let's stick to the standard :). I suggest:

```html
		<label class="group-name" for="yt_content_none"><?php echo _t('ext.yt_videos.content.title'); ?></label>
		<div class="group-controls">
			<label for="yt_content_none">
				<input type="radio" id="yt_content_none" name="yt_content_choice" value="none" <?= $this->getContentHandling() === 'none' ? 'checked' : '' ?>>
				<?php echo _t('ext.yt_videos.content.none'); ?>
			</label>
			<label for="yt_content_append">
				<input type="radio" id="yt_content_append" name="yt_content_choice" value="append" <?= $this->getContentHandling() === 'append' ? 'checked' : '' ?>>
				<?php echo _t('ext.yt_videos.content.append'); ?>
			</label>
			<label for="yt_content_format">
				<input type="radio" id="yt_content_format" name="yt_content_choice" value="format" <?= $this->getContentHandling() === 'format' ? 'checked' : '' ?>>
				<?php echo _t('ext.yt_videos.content.format'); ?>
			</label>
		</div>
```

Reply:
Yes, good point. Fixed (and tested).